lib/commands/log.js

```diff
@@ -98,23 +111,30 @@ export default {
         sim: this.device,
         udid: this.isRealDevice() ? this.opts.udid : undefined,
       });
-
-      if (this.isRealDevice()) {
-        this.logs.syslog = new IOSDeviceLog({
+      this.logs.syslog = this.isRealDevice()
+        ? new IOSDeviceLog({
           udid: this.opts.udid,
           showLogs: this.opts.showIOSLog,
           log: this.log,
-        });
-      } else {
-        this.logs.syslog = new IOSSimulatorLog({
+        })
+        : new IOSSimulatorLog({
           sim: /** @type {import('appium-ios-simulator').Simulator} */ (this.device),
           showLogs: this.opts.showIOSLog,
           iosSimulatorLogsPredicate: this.opts.iosSimulatorLogsPredicate,
           log: this.log,
         });
+      if (_.isBoolean(this.opts.showSafariConsoleLog)) {
+        this.logs.safariConsole = new SafariConsoleLog({
+          showLogs: this.opts.showSafariConsoleLog,
```

Comment:
Could you please explain what is the disadvantage of the current logic?

showSafariConsoleLog
  - undefined: no logs are collected, nothing is shown in server logs
  - false:  logs are collected, nothing is shown in server logs
  - true: logs are collected and shown in server logs

Reply:
Oh, I missed the `false` case. I thought bool, so false/undefined or true like `showiOSLog` behavior. Let me check again

Reply:
Correct, `false` is what I meant.

It looks like this title should be `feat:` since we introduced two caps to change the existing behavior. I also wondered if it would be helpful to address `false` is the same behavior xcuitest driver v7.21 and lower.

Reply:
changed to feat:, Updated docs